cypresse2e/cypress/integration/e2eTests/offerHelp/offerHelpQuestion.spec.js

```diff
@@ -0,0 +1,109 @@
+            offerHelpQuestion.getQuestionOne().should('be.visible').contains(questionOne);
+        });
+
+        it('Offer Help Question One page contains answer options & checkboxes to select option/options for question one', () => {
+            var checkboxOptionOne = "As a Volunteer";
+            var checkboxOptionTwo = "As a Donor/Investor"
+            var checkboxOptionThree = "As an Organisation"
+            var checkboxOne
+            var checkboxTwo
+            var checkboxThree
+            offerHelpQuestion.getCheckboxOptionOne().should('be.visible').contains(checkboxOptionOne);
+            offerHelpQuestion.getCheckboxOptionTwo().should('be.visible').contains(checkboxOptionTwo);
+            offerHelpQuestion.getCheckboxOptionThree().should('be.visible').contains(checkboxOptionThree);
+            offerHelpQuestion.getCheckboxOne().should('be.visible').check({ force: true }).should('be.checked');
+            offerHelpQuestion.getCheckboxTwo().should('be.visible').check({ force: true }).should('be.checked');
+            offerHelpQuestion.getCheckboxThree().should('be.visible').check({ force: true }).should('be.checked');
+        });
+
+        it('Offer Help Question One Back Button is visible and clickable', () => {
+            var backButton = offerHelpQuestion.getBackButtonOne();
+            backButton.should('be.visible').click();
+
+        });
+
+
+        it('Offer Help Question One Next Button is visible and clickable and Offer Help Question Two Page loads successfully', () => {
+            var nextButton = offerHelpQuestion.getNextButtonOne();
+            nextButton.should('be.visible').click();
+        });
+
+    });
+
+    context('User goes to offer help question 2', () => {
+        beforeEach(() => {
+            offerHelpQuestion.visitOfferHelpTwo();
+        });
+
+        it('Offer Help Question Two Page contains heading & question two', () => {
+            var questionNumberTwo = "Question 2/3";
+            var questionTwo = "Where are you located?";
+            var infoText = "We want to show you the most relevant results";
+            offerHelpQuestion.getQuestionNumberTwo().should('be.visible').contains(questionNumberTwo);
+            offerHelpQuestion.getQuestionTwo().should('be.visible').contains(questionTwo);
+            offerHelpQuestion.getInfoText().should('be.visible').contains(infoText);
+
+        });
+
+        it('Offer Help Question Two Page - location can be typed and chosen from a list of options from the typeahead dynamic dropdown', () => {
+            var locationInputLine = offerHelpQuestion.getLocationInputLine();
+            locationInputLine.should('be.visible').click();
+            locationInputLine.type("Los")
+            offerHelpQuestion.getLocationDropDown().should('be.visible');
+            offerHelpQuestion.getLocationDropDown().find('div.ant-select-item-option').contains('Los Gatos, CA, USA').click();
+        });
```

Comment:
I'm getting an error running this code block. For me, I'm seeing a different location in the drop down list when I execute the test. 

![image](https://user-images.githubusercontent.com/56090896/93359443-9ddea200-f810-11ea-938c-66f4441a9720.png)

![image](https://user-images.githubusercontent.com/56090896/93359515-b2bb3580-f810-11ea-84b1-e3349c2b32a5.png)

Since the order of results aren't always consistent, I suggest trying a similar code to the one I wrote for the Request Help page instead:

```
it('User can type in location field and select location from the list', () => {
            var locationField = requestHelpQuestion.getRhLocationField();
            locationField.should('be.visible').click();
            locationField.type(LOCATION)
                .and('have.attr', 'aria-expanded', 'true')
                .and('have.attr', 'aria-activedescendant', 'RES2_EAZ_list_0').click();
        });
```

Reply:
It was running fine on my end, was able to check and select a specified location but I get your point on the location options being inconsistent. Will work on this too. Thanks for the feedback.